apps/ejabberd/test/auth_external_SUITE.erl

```diff
@@ -24,7 +24,7 @@ all_tests() ->
     ].
 
 init_per_suite(C) ->
-    ok = application:ensure_started(stringprep),
```

Comment:
I don't fully remember but probably because of `rebar3 ct` and it being more vulnerable. It causes test cases to fail in places where `rebar ct` didn't.
@erszcz do you remember why such change? this was introduced in this commit: https://github.com/esl/MongooseIM/commit/5ac4b9856cc0fd22dd7a0f6eec963edba12b6427